.github/workflows/build.yml

```diff
@@ -8,166 +8,158 @@ on:
       - "v*"
 jobs:
   linux:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```

Comment:
I think we want to keep this on 16.04 if at all possible.  I can't run the current version on my laptop running ubuntu 18.04.  I get this:

```[9974] Error loading Python lib '/home/lex/.config/inkscape/extensions.binary/inkstitch/bin/libpython3.9.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /home/lex/.config/inkscape/extensions.binary/inkstitch/bin/libpython3.9.so.1.0)```

Reply:
@rumpeltux Just wanted to make sure you saw this.  I'm going to switch it back to 16.04 now.  Because we're using pyinstaller, this just sets 16.04 as the _minimum_ Ubuntu version needed to run Ink/Stitch.